doc/changes.rst

```diff
@@ -1,6 +1,113 @@
 Changelog
 =========
 
+Release v0.3.0 (2021/02/15)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4522136.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4522136
+
+Highlights
+
+* 🎉 **Third minor release of PyGMT** 🎉
+* Wrap inset (#788) for making overview maps and subplot (#822) for multi-panel figures
+* Apply standardized formatting conventions (#775) across most documentation pages
+* Drop Python 3.6 support (#699) so PyGMT now requires Python 3.7 or newer
+
+New Features
+
+* Wrap grd2cpt (#803)
+* Let Figure.text() support record-by-record transparency (#716)
+* Provide basic support for FreeBSD (#700, #878)
+
+Enhancements
+
+* Let load_earth_relief() support the 'region' parameter for all resolutions (#873)
+* Improve how PyGMT finds the GMT library (#702)
+* Add common alias panel (-c) to all plotting functions (#853)
+* Add aliases dcw (#765) and lakes (#781) to Figure.coast
+* Add alias shading to Figure.colorbar (#752)
+* Add alias annotation (A) to Figure.contour() (#883)
+* Wrap Figure.grdinfo() aliases (#799)
+* Add aliases frame and cmap to Figure.colorbar() (#709)
+* Add alias frame to Figure.grdview() (#707)
+* Improve the error message when PyGMT fails to load the GMT library (#814)
+* Add GMTInvalidInput Error for Figure.coast (#787)
```

Comment:
Consistently use `Figure.function()` (with parentheses） or `Figure.function` (without parentheses). 
```suggestion
* Add aliases dcw (#765) and lakes (#781) to Figure.coast()
* Add alias shading to Figure.colorbar() (#752)
* Add alias annotation (A) to Figure.contour() (#883)
* Wrap Figure.grdinfo() aliases (#799)
* Add aliases frame and cmap to Figure.colorbar() (#709)
* Add alias frame to Figure.grdview() (#707)
* Improve the error message when PyGMT fails to load the GMT library (#814)
* Add GMTInvalidInput error to Figure.coast() (#787)
```

Reply:
I think without parentheses is better.

Reply:
I'm OK with both. Feel free to make any changes.

Reply:
There's still a lot of parentheses in the changelog @willschlitzer, could you remove all of them (just the v0.3.0 ones) to be consistent?